### PR TITLE
[13.0][MIG] purchase_order_line_sequence

### PR DESCRIPTION
**Pull request by DavidJForgeFlow**
Migration to 13.0

@ForgeFlow

### CODE REVIEW

File: purchase_order_line_sequence/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Purchase Order Line Sequence",
+    "summary": "Adds sequence to PO lines and propagates it to"
+    "Invoice lines and Stock Moves",
+    "version": "13.0.1.0.0",
+    "category": "Purchase Management",
+    "author": "Camptocamp, "
+    "Eficent, "
```
Comment (mariadforgeflow) [inline purchase_order_line_sequence/__manifest__.py]:
```suggestion
    "ForgeFlow, "
```
It should be changed everywhere:)

Comment (DavidJForgeFlow) [reply purchase_order_line_sequence/__manifest__.py]: Done!

**Review by mariadforgeflow — APPROVED** [review]
LGTM!

**Review by LoisRForgeFlow — APPROVED** [review]
Code review LGTM

**Pull request merged**